gfx/layers/client/ImageClient.cpp

```diff
@@ -170,6 +170,7 @@ bool ImageClientSingle::UpdateImage(ImageContainer* aContainer,
   if (mLastUpdateGenerationCounter == generationCounter) {
     return true;
   }
+  recordreplay::RecordReplayAssert("[RUN-1252] ImageClientSingle::UpdateImage #1");
```

Comment:
It'd be nice to start using the `[RUN-1252-1357]` syntax

Reply:
> It'd be nice to start using the `[RUN-1252-1357]` syntax

Let's add an issue in the crash observability followups project to first recognize this pattern.  I don't want to start using it until we can make sure our auto-buckets can properly catch it and currently they don't.

Reply:
@jasonLaster We have to fix this first: https://linear.app/replay/issue/SCS-784/[crash-buckets]-allow-more-flexible-labels

Reply:
👍